docs/plugins.rst

```diff
@@ -199,16 +190,6 @@ definitions in Airflow.
         @expose("/")
         def test(self):
             return self.render("test_plugin/test.html", content="Hello galaxy!")
-    v_appbuilder_view = TestAppBuilderBaseView()
-    v_appbuilder_package = {"name": "Test View",
-                            "category": "Test Plugin",
-                            "view": v_appbuilder_view}
-
-    # Creating a flask appbuilder Menu Item
-    appbuilder_mitem = {"name": "Google",
-                        "category": "Search",
-                        "category_icon": "fa-th",
-                        "href": "https://www.google.com"}
 
     # Defining the plugin class
     class AirflowTestPlugin(AirflowPlugin):
```

Comment:
This file is already in the repository. Can you add a reference to this file using the literalinclude directive? Or even better - you can use the exampleinclude directive to add a button to the full source code
https://github.com/apache/airflow/blob/d333ffa5cc4f79ccccf36117d4b33b546da9091e/tests/plugins/test_plugin.py